packages/devtools/lib/src/inspector/inspector.dart

```diff
@@ -96,8 +105,19 @@ class InspectorScreen extends Screen {
     return screenDiv;
   }
 
+  @override
+  void entering() {
+    if (inspectorMessages.isNotEmpty) {
```

Comment:
Should the `Framework` class or the `Screen` class be responsible for pushing and popping messages on page entry and exit, instead of each individual page? We do something like for the status line contributions.

Reply:
Using MessageManager to push and pop messages at the appropriate times.